decksite/templates/cards.mustache

```diff
@@ -1,4 +1,8 @@
 {{> banner}}
 <section>
+    <button id="toggle-filters-button" onclick="PD.filter.toggleDisplayFilter()">Show filters</button>
```

Comment:
Please attach the onclick behavior in pd.js don’t inline it here. It’s our standard.